mmgen/models/gans/pix2pix.py

```diff
@@ -1,340 +1,181 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-import os.path as osp
 
-import mmcv
-import numpy as np
 import torch
+from torch.nn.parallel.distributed import _find_tensors
 
-from mmgen.models.builder import MODELS, build_module
-from mmgen.models.misc import tensor2img
+from mmgen.models.builder import MODELS
 from ..common import set_requires_grad
-from .base_gan import BaseGAN
+from .static_translation_gan import StaticTranslationGAN
```

Comment:
You can create a new folder `mmgen/models/translation_models` and tidy related files.